plugins/processors/awsappsignals/config/resolvers_test.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewEKSResolver(t *testing.T) {
+	resolver := NewEKSResolver("test")
```

Comment:
nit: missing ClusterName validation

Reply:
The validation is in the `config.go`. The test is covered [here](https://github.com/aws/amazon-cloudwatch-agent/pull/980/files#diff-e8991560630a53c21e6ef766fe56428ca993d3dc9ebec3f00595618af2a57f1eR28).